core/profiles/standard/standard.install

```diff
@@ -346,14 +346,27 @@ function standard_install() {
 
   // Enable default permissions for system roles.
   $filtered_html_permission = filter_permission_name($filtered_html_format);
-  user_role_grant_permissions(BACKDROP_ANONYMOUS_ROLE, array('access content', 'access comments', $filtered_html_permission));
-  user_role_grant_permissions(BACKDROP_AUTHENTICATED_ROLE, array('access content', 'access comments', 'post comments', 'skip comment approval', 'upload editor images', $filtered_html_permission));
+  $anonymous_permissions = array(
+    'access content',
+    'access comments',
+    $filtered_html_permission
```

Comment:
Coding standards dictate that we add a comma after the last entry in arrays, when they are in this format:

```suggestion
    $filtered_html_permission,
```